tests/test_cortex_endpoint.py

```diff
@@ -0,0 +1,36 @@
+from thehive4py.client import TheHiveApi
+
+
+class TestCortexEndpoint:
+    def test_list_analyzers(self, thehive: TheHiveApi):
+        analyzers = thehive.cortex.list_analyzers()
+        assert isinstance(analyzers, list)
+        for analyzer in analyzers:
+            assert "name" in analyzer.keys()
+            assert "id" in analyzer.keys()
```

Comment:
Let's keep it even simpler:

```suggestion
        assert analyzers == []
```